pkg/sessionctx/variable/session.go

```diff
@@ -792,6 +792,9 @@ type SessionVars struct {
 	// LastFoundRows is the number of found rows of last query statement
 	LastFoundRows uint64
 
+	// LastRUConsumption is the total RU consumption of last query statement
+	LastRUConsumption float64
```

Comment:
Here is an independent variable like `sessVars.TxnCtx.StartTS`, but I reuse the user-aware variable `last_query_info` to collect them with some condition checks. I think it can keep the consist with those conditions.